File: java/src/main/java/javaprojet/Entity/MenuBurger.java
package javaprojet.Entity;

public class MenuBurger {
    private int id;
    private int menuId;
    private int burgerId;
    private int quantite;

    public MenuBurger() {
    }

    public MenuBurger(int id, int menuId, int burgerId, int quantite) {
        this.id = id;
        this.menuId = menuId;
        this.burgerId = burgerId;
        this.quantite = quantite;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getMenuId() {
        return menuId;
    }

    public void setMenuId(int menuId) {
        this.menuId = menuId;
    }

    public int getBurgerId() {
        return burgerId;
    }

    public void setBurgerId(int burgerId) {
        this.burgerId = burgerId;
    }

    public int getQuantite() {
        return quantite;
    }

    public void setQuantite(int quantite) {
        this.quantite = quantite;
    }

    @Override
    public String toString() {
        return "MenuBurger{" +
                "id=" + id +
                ", menuId=" + menuId +
                ", burgerId=" + burgerId +
                ", quantite=" + quantite +
                '}';
    }
}

File: Symfony/BrasilBurgerGest/src/Entity/MenuBurger.php
<?php

namespace App\Entity;

use App\Repository\MenuBurgerRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: MenuBurgerRepository::class)]
class MenuBurger
{
    #[ORM\Id, ORM\GeneratedValue, ORM\Column(type:"bigint")]
    private ?int $id = null;

    #[ORM\ManyToOne(targetEntity:Menu::class, inversedBy:"menuBurgers")]
    #[ORM\JoinColumn(nullable:true)]
    private ?Menu $menu = null;

    #[ORM\ManyToOne(targetEntity:Burger::class, inversedBy:"menuBurgers")]
    #[ORM\JoinColumn(nullable:true)]
    private ?Burger $burger = null;

    #[ORM\Column(type:"integer")]
    private ?int $quantite = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getMenu(): ?Menu
    {
        return $this->menu;
    }

    public function setMenu(?Menu $menu): static
    {
        $this->menu = $menu;

        return $this;
    }

    public function getBurger(): ?Burger
    {
        return $this->burger;
    }

    public function setBurger(?Burger $burger): static
    {
        $this->burger = $burger;

        return $this;
    }

    public function getQuantite(): ?int
    {
        return $this->quantite;
    }

    public function setQuantite(int $quantite): static
    {
        $this->quantite = $quantite;

        return $this;
    }
}
